test/utils.py

```diff
@@ -71,3 +74,30 @@ def bell_in_qobj(backend: IBMQBackend, shots: int = 1024) -> Qobj:
     """
     return assemble(transpile(ReferenceCircuits.bell(), backend=backend),
                     backend=backend, shots=shots)
+
+
+def cancel_job(job: IBMQJob, verify: bool = False) -> bool:
+    """Cancel a job.
+
+    Args:
+        job: Job to cancel.
+        verify: Verify job status.
+
+    Returns:
+        Whether job has been cancelled.
+    """
+    cancelled = False
+    for _ in range(2):
+        # Try twice in case job is not in a cancellable state
+        try:
+            if job.cancel():
```

Comment:
You forgot to set `cancelled = True` within `if job.cancel()` 😅

Reply:
Oops